### PR TITLE
Update supported-environments.md

### PR DESCRIPTION
**Pull request by toco-cam**
This table seems redundant to https://docs.camunda.org/enterprise/download/#production-distribution-without-elasticsearch

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by akeller — APPROVED** [review]
I ultimately don't care where this info sits, but for some additional context, we moved content from docs.camunda.org to docs.camunda.io for visibility and took a "Camunda 8 first" approach. Routing people to docs.camunda.org may feel counter to that C8 first approach.

Given the SEO improvements in the C7 docs, the visibility is not a challenge anymore. However, the DevEx team does very limited maintenance and updates to the C7 docs. If you and the Optimize team are ok to update this on your own, let's merge this.

File: docs/reference/supported-environments.md [hunk 1]
```diff
@@ -77,21 +77,4 @@ You can also use newer versions of Desktop and Web Modeler with older Zeebe vers
...
-| Optimize 3.9.x  | Camunda 7.16.x, 7.17.x, 7.18.x         | OpenJDK 11+               | 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+                                   |
-| Optimize 3.10.x | Camunda 7.17.x, 7.18.x, 7.19.x         | OpenJDK 17+               | 7.16.2+, 7.17.0+, 8.5.0+, 8.6.0+                                              |
-| Optimize 3.11.x | Camunda 7.17.x, 7.18.x, 7.19.x, 7.20.x | OpenJDK 17+               | 8.7.0+, 8.8.0+                                                                |
-
-:::note Elasticsearch support
-[Elastic's Elasticsearch](https://www.elastic.co/elasticsearch/) is the only supported version of Elastic compatible with Optimize.
-
-AWS OpenSearch is not supported at this time.
-:::
+See https://docs.camunda.org/enterprise/download/#camunda-optimize
```
Comment (akeller) [inline docs/reference/supported-environments.md:80]:
```suggestion
See https://docs.camunda.org/enterprise/download/#camunda-optimize.

```

**Review by pepopowitz — APPROVED** [review]

**Comment by christinaausley** [comment]
Hi @toco-cam -- you'll need to sign the CLA one more time 👍

**Comment by akeller** [comment]
Merging this PR will only add these changes to `/next/`. If you would like to add these changes to the latest version, you'll need to update the same file in `version-8.3` (this is not an Optimize-specific file).

**Comment by toco-cam** [comment]
All good - this is not super urgent

**Pull request merged**